### PR TITLE
[CSL-1436] Make blkSecurityParam flexible, remove travis mode

### PR DESCRIPTION
**Pull request by gromakovsky**
The PR is WIP, it was started by me and will be finished by @pva701.

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -12,7 +12,7 @@ matrix:
 before_install:
 - echo $TRAVIS_OS_NAME
 - export CSL_SYSTEM_TAG=$TRAVIS_OS_NAME
-- export DCONFIG=travis
+- export DCONFIG=qanet_tns_wallet
```
Comment (domenkozar) [inline .travis.yml]: Please let's agree on constant conventions, this will break things

Comment (gromakovsky) [inline .travis.yml]: This issue and PR are only about making some constants flexible which allows us to get rid of `travis` mode. It's the first step to using new conventions. After it's done we will be able to easily change these names (but in another issue/PR).

Comment (domenkozar) [inline .travis.yml]: k

File: core/Pos/Core/Context.hs [hunk 2]
```diff
@@ -9,10 +16,27 @@ module Pos.Core.Context
...
+import           Control.Lens     (Getter, makeLenses)
+
 import           Pos.Core.Address (addressHash, makePubKeyAddress)
-import           Pos.Core.Types   (Address, StakeholderId)
+import           Pos.Core.Types   (Address, BlockCount, StakeholderId)
 import           Pos.Crypto       (PublicKey, SecretKey, toPublic)
 
+-- | Core constants. They should be really constant and never change.
+data CoreConstants = CoreConstants
+    { _ccBlkSecuriryParam :: !BlockCount
```
Comment (pva701) [inline core/Pos/Core/Context.hs]: >securiry
ri-ri-ri-ri-ri
cc @volhovM

**Pull request closed without merging**